lib/twingly/amqp/default_exchange_publisher.rb

```diff
@@ -3,12 +3,28 @@ module AMQP
     class DefaultExchangePublisher
       include Publisher
 
+      DEFAULT_EXCHANGE = ""
+
       def initialize(queue_name:, connection: nil)
         options.routing_key = queue_name
 
         connection ||= Connection.instance
         @exchange = connection.create_channel.default_exchange
       end
+
+      def self.delayed(delay_queue_name:, delay_ms:, target_queue_name:, connection: Connection.instance)
```

Comment:
I liked the ordering of arguments in the README:

```suggestion
      def self.delayed(delay_queue_name:, target_queue_name:, delay_ms:, connection: Connection.instance)
```